castervoice/rules/apps/chat/MSTeamsRule.py

```diff
@@ -0,0 +1,119 @@
+from castervoice.lib.actions import Key, Text, Mouse
+from castervoice.lib.ctrl.mgr.rule_details import RuleDetails
+from castervoice.lib.merge.additions import IntegerRefST
+from castervoice.lib.merge.state.short import R
+from dragonfly import (AppContext, Choice, Dictation, Function, MappingRule,
+                       Repeat)
+
+class MSTeamsRule(MappingRule):
+    name = "microsoft teams"
```

Comment:
`name` isn't required here only in RuleDetails for MappingRule's.

Reply:
Thanks that's good to know, i've got a bunch of other commands now that i'm using, is there a reason to why windows key bindings are pretty light? i'm not sure if others will find them useful but i have definitely have. E.g. 
        "open explorer":
            R(Key("w-e")),
        "show desktop":
            R(Key("w-d")),
        "windows speech":
            R(Key("w-ctrl")),
        "admin tools":
            R(Key("w-x")),
        "windows run":
            R(Key("w-r")),
        "press windows":
            R(Key("win")),
        "windows search":
            R(Key("w-s")),
        "taskbar":
            R(Key("w-t")),
        "taskbar <n>":
            R(Key("w-%(n)s")),
        "taskbar menu <n>":
            R(Key("wa-%(n)s")),

Reply:
The Windows keyboard bindings could be added.